### PR TITLE
Extract the bases used for the commits

### LINKED ISSUE #53: Extract EidNym generators and Curve
At present, there is no method to extract the generators used in computing `EidNym` from the issuer's public key. These generators could potentially be utilized in creating other ZK proofs involving EidNym.

This GitHub issue proposes implementing a mechanism to extract these generators. One potential solution would be to receive the generators during the key import process. This could be achieved by augmenting the IdemixIssuerPublicKeyImportOpts struct with an additional field that carries the generators.

Same reasoning is for the elliptic curve used.

### PR DESCRIPTION
**Pull request by ale-linux**
This PR adds support to retrieve the bases used for the commitments generated by the scheme.

Resolves #53

### CODE REVIEW

File: bccsp/types/idemixopts.go [hunk 1]
```diff
@@ -44,11 +44,38 @@ func (o *IdemixIssuerKeyGenOpts) Ephemeral() bool {
...
 // IdemixIssuerPublicKeyImportOpts contains the options for importing of an Idemix issuer public key.
 type IdemixIssuerPublicKeyImportOpts struct {
 	Temporary bool
 	// AttributeNames is a list of attributes to ensure the import public key has
 	AttributeNames []string
+	// CommitmentBasesRequest describes the request for commitment bases
+	CommitmentBasesRequest CommitmentBasesRequest
+	// RhIndex is the index of attribute containing the revocation handler.
+	RhIndex int
+	// EidIndex contains the index of the EID attrbiute
```
Comment (adecaro) [inline bccsp/types/idemixopts.go:73]: attribute

**Review by adecaro — APPROVED** [review]
LGTM

File: bccsp/schemes/aries/issuer.go [hunk 2]
```diff
@@ -58,6 +58,32 @@ type Issuer struct {
 	Curve *math.Curve
 }
 
+func (i *Issuer) Bases(key types.IssuerPublicKey, ipkType types.CommitmentBasesRequest, RhIndex, EidIndex, SKIndex int) (map[types.CommitmentType]interface{}, error) {
```
Comment (adecaro) [inline bccsp/schemes/aries/issuer.go:61]: please, add a comment here describing what will be returned

**Review by adecaro — APPROVED** [review]
LGTM

File: bccsp/types/idemixopts.go (hunk 1 above)
Comment (ale-linux) [reply bccsp/types/idemixopts.go]: done, thanks!

File: bccsp/schemes/aries/issuer.go (hunk 2 above)
Comment (ale-linux) [reply bccsp/schemes/aries/issuer.go:64]: done, thanks!

**Review by adecaro — APPROVED** [review]
LGTM

**Pull request merged**